google/devtools/clouddebugger/v2/clouddebugger_gapic.yaml

```diff
@@ -0,0 +1,162 @@
+type: com.google.api.codegen.ConfigProto
+language_settings:
+  java:
+    package_name: com.google.cloud.devtools.clouddebugger.spi.v2
+  python:
+    package_name: google.cloud.gapic.devtools.clouddebugger.v2
```

Comment:
By making this change upstream, you mean changing the proto package from `google.devtools.clouddebugger` to `google.debugger.v2`, right? The main reason I'm aware of for not trying to do this is just pragmatics, but I don't actually know what sort of work/coordination would be required to make this happen...